kernel/src/hil/buzzer.rs

```diff
@@ -0,0 +1,37 @@
+//! Interface for buzzer use.
+
+use crate::ErrorCode;
+
+pub trait BuzzerClient {
+    /// Called when the current sound played by the buzzer has finished
+    /// or it was stopped.
+    fn buzzer_done(&self, status: Result<(), ErrorCode>);
+}
+
+pub trait Buzzer<'a> {
```

Comment:
Please add a comment to explain that the main purpose of the Buzzer HIL is. (ex: playing a sound of a fixed frequency).